scripts/tools/DeliveryExecutionReport.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoSystemStatus\scripts\tools;
+
+use oat\oatbox\extension\AbstractAction;
+use oat\oatbox\log\LoggerAwareTrait;
+use oat\tao\model\taskQueue\TaskLog;
+use oat\tao\model\taskQueue\TaskLog\Broker\TaskLogBrokerInterface;
+use oat\tao\model\taskQueue\TaskLog\TaskLogFilter;
+use oat\taoAct\model\Ods\DeliveryLogCacheService;
+use oat\taoDelivery\model\execution\ServiceProxy;
+use oat\taoProctoring\model\deliveryLog\DeliveryLog;
+use oat\taoProctoring\model\monitorCache\DeliveryMonitoringService;
+use common_report_Report as Report;
+use common_ext_ExtensionsManager;
+
+/**
+ * Class DeliveryExecutionReport
+ *
+ * @package oat\taoSystemStatus\scripts\tools
+ * @author Andrei Niahrou, <Andrei.Niahrou@1pt.com>
+ */
+class DeliveryExecutionReport extends AbstractAction
+{
+    use LoggerAwareTrait;
+
+    private static $validMethods = ['show'];
+
+    /** @var array */
+    private $params;
+
+    /** @var string */
+    private $method;
+
+    /**
+     * @inheritdoc
+     */
+    public function __invoke($params = [])
+    {
+        $this->init($params);
```

Comment:
Let's reduce the complexity and assume that this action will have only one responsibility: get delivery execution data.
So let's get rid of checking `$validMethods` and show function.